src/main/resources/assets/js/components/TaskListComponent.jsx

```diff
@@ -102,14 +103,14 @@ var TaskListComponent = React.createClass({
               if (!isActive) { allTasksSelected = false; }
 
               return (
-                  <TaskListItemComponent
-                    isActive={isActive}
-                    key={task.id}
-                    taskHealthMessage={this.props.formatTaskHealthMessage(task)}
-                    onToggle={this.props.onTaskToggle}
-                    onTaskDetailSelect={this.props.onTaskDetailSelect}
-                    hasHealth={hasHealth}
-                    task={task} />
+                <TaskListItemComponent
```

Comment:
Would you mind getting rid of the same if statements here?

You should probably end up with something like this:
```
getTasks: function () {
  return (
    this.props.tasks.map(function (task) {
        // Expicitly check for Boolean since the key might not exist in the
        // object.
        var isActive = this.props.selectedTasks[task.id] === true;
        if (!isActive) { allTasksSelected = false; }

        return (
          <TaskListItemComponent
            appId={appId}
            hasHealth={hasHealth}
            isActive={isActive}
            key={task.id}
            onToggle={this.props.onTaskToggle}
            task={task}
            taskHealthMessage={this.props.formatTaskHealthMessage(task)}/>
        );
      }, this)
  );
}

render: function () {
  ...
  <tbody className={noTasksClassSet}>
    <tr >
      <td className="text-center" colSpan="7">
        No tasks running.
      </td>
    </tr>
  </tbody>
  <tbody className={loadingClassSet}>
    <tr>
      <td className="text-center text-muted" colSpan="7">
        Loading tasks...
      </td>
    </tr>
  </tbody>
  <PagedContentComponent
      currentPage={this.props.currentPage}
      itemsPerPage={this.props.itemsPerPage}
      tag="tbody" >
    <tr classNam={hasErrorClassSet}>
      <td className="text-center text-danger" colSpan="7">
        Error fetching tasks. Refresh the list to try again.
      </td>
    </tr>
    {this.getTaskNodes()}
  </PagedContentComponent>
```